### DISCUSSION TITLE
My YOLO and Pull Shark achievements on my profile are wrong and I would like it to be removed

### DISCUSSION THREAD

**Discussion by arakshit-tibco** [community/community#28806] [Other Feature Feedback, Questions, & Ideas]
The PRs that are mentioned explaining both badges YOLO and Pull Shark clearly shows that I did not merge the PR without the approval and the PR was not created for a merged branch.

These badges are incorrect and technically portrays wrong information on my profile. Please help and remove these badges.

Thanks!

**Comment by radar-blip** [comment]
You can disable them in your settings if that would help. Some of them are kinda buggy it seems.

![show_achievements](https://user-images.githubusercontent.com/39196767/183522272-b20813b5-9c2f-4c56-a498-23979ae7ee7b.png)

**Reply by arakshit-tibco** [reply]
thanks @linuxlawson for responding. I would not want to disable all badges on my profile but just those buggy ones. @github-staff should fix the logic of how they evaluate these incorrect badges. Thanks!

**Reply by arakshit-tibco** [reply]
will keep this question unanswered until this has a proper fix. Considering we dont have a concrete answer for this, hope you wont mind @linuxlawson

**Reply by radar-blip** [reply]
Nope, I dont mind at all. 
Here is more info on the achievement badges if you want to take a look at it.

[Achievement-Badges](https://github.com/drknzz/GitHub-Achievements)

**Comment by queenofcorgis** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by airtower-luna** [comment]
> the PR was not created for a merged branch.

I think you misunderstand the description of the "Pull Shark". It means you have created PRs that have then been merged. Which is usually the goal when creating a PR. :slightly_smiling_face:

**Comment by NikhilMath** [comment]
So your badges are actually correct and are displaying as they're indended to be. 
You are slightly confused on what they are, but they are right. trust me. I had the same mistake.
